### PR TITLE
bpo-44740: Lowercase "internet" and "web" where appropriate.

### PR DESCRIPTION
**Pull request by felixxm**
[bpo-44740](https://bugs.python.org/issue44740)

https://bugs.python.org/issue44740

### CODE REVIEW

**Comment by ambv** [comment]
As I expected, docs between `main` and `3.9` are too divergent to cleanly backport this manually. @felixxm, if you'd like to see your change land in 3.9 docs too, we will need a fresh PR against the `3.9` branch.

File: Doc/howto/ipaddress.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ A Note on IP Versions
 ---------------------
 
 For readers that aren't particularly familiar with IP addressing, it's
-important to know that the Internet Protocol is currently in the process
+important to know that the internet protocol is currently in the process
```
Comment (terryjreedy) [inline Doc/howto/ipaddress.rst:35]: I am not sure about this one.

File: Doc/library/ftplib.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ This module defines the class :class:`FTP` and a few related items. The
 this to write Python programs that perform a variety of automated FTP jobs, such
 as mirroring other FTP servers.  It is also used by the module
 :mod:`urllib.request` to handle URLs that use FTP.  For more information on FTP
-(File Transfer Protocol), see Internet :rfc:`959`.
+(File Transfer Protocol), see internet :rfc:`959`.
```
Comment (terryjreedy) [inline Doc/library/ftplib.rst:20]: Not sure because not sure why 'Internet' is present in reference.

File: Doc/library/http.server.rst [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 --------------
 
-This module defines classes for implementing HTTP servers (Web servers).
+This module defines classes for implementing HTTP servers (web servers).
```
Comment (terryjreedy) [inline Doc/library/http.server.rst:17]: `(W/web servers)` not really needed.

File: Mac/BuildScript/resources/License.rtf [hunk 4]
```diff
@@ -106,7 +106,7 @@ BEOPEN PYTHON OPEN SOURCE LICENSE AGREEMENT VERSION 1\
...
+2. Subject to the terms and conditions of this License Agreement, CNRI hereby grants Licensee a nonexclusive, royalty-free, world-wide license to reproduce, analyze, test, perform and/or display publicly, prepare derivative works, distribute, and otherwise use Python 1.6.1 alone or in any derivative version, provided, however, that CNRI's License Agreement and CNRI's notice of copyright, i.e., "Copyright (c) 1995-2001 Corporation for National Research Initiatives; All Rights Reserved" are retained in Python 1.6.1 alone or in any derivative version prepared by Licensee. Alternately, in lieu of CNRI's License Agreement, Licensee may substitute the following text (omitting the quotes): "Python 1.6.1 is made available subject to the terms and conditions in CNRI's License Agreement. This Agreement together with Python 1.6.1 may be located on the internet using the following unique, persistent identifier (known as a handle): 1895.22/1013. This Agreement may also be obtained from a proxy server on the internet using the following URL: http://hdl.handle.net/1895.22/1013".\
```
Comment (terryjreedy) [inline Mac/BuildScript/resources/License.rtf:109]: I cannot see what changed.

**Review by terryjreedy — COMMENTED** [review]
Changes are formally correct.

@JulienPalard Change is based on 2016 revised Associated Press Style Book.  'web' versus 'Web' looks correct to me.  I am not sure whether any 'Internet' (versus 'internet')  uses should remain.  Your opinion?

@ambv I subsequently saw that this is merged already.  Fine with me since you think so.  Is there any situation in which 'Internet' is still proper, is has it been completely genericized?

File: Mac/BuildScript/resources/License.rtf (hunk 4 above)
Comment (felixxm) [reply Mac/BuildScript/resources/License.rtf:109]: `the internet` is twice in this line.

File: Doc/library/ftplib.rst (hunk 2 above)
Comment (felixxm) [reply Doc/library/ftplib.rst:20]: I was thinking about removing `internet` from this sentence, but the `internet RFC-*` pattern is used few times in docs. I can send subsequent PR.

**Comment by felixxm** [comment]
@ambv Thanks :star:  

> As I expected, docs between `main` and `3.9` are too divergent to cleanly backport this manually. @felixxm, if you'd like to see your change land in 3.9 docs too, we will need a fresh PR against the `3.9` branch.

I wasn't sure what is the policy of backporting docs changes. It's probably not worth the extra work for documentation translators :thinking:

File: Doc/howto/ipaddress.rst (hunk 1 above)
Comment (serhiy-storchaka) [reply Doc/howto/ipaddress.rst:35]: It is a proper name. Usually used as abbreviations IPv4 and IPv6. I think it should be titlecased.

File: Doc/library/ftplib.rst (hunk 2 above)
Comment (serhiy-storchaka) [reply Doc/library/ftplib.rst:20]: I think it is an adjective here and it is correct to lowercase it.

File: Doc/whatsnew/2.3.rst [hunk 5]
```diff
@@ -672,7 +672,7 @@ with older versions of the Distutils::
    from distutils import core
    kw = {'name': "Quixote",
          'version': "0.5.1",
-         'description': "A highly Pythonic Web application framework",
+         'description': "A highly Pythonic web application framework",
```
Comment (serhiy-storchaka) [inline Doc/whatsnew/2.3.rst:675]: It is a description of an external package. https://pypi.org/project/Quixote/
"Web" is used in title case there.

File: Mac/BuildScript/resources/License.rtf (hunk 4 above)
Comment (serhiy-storchaka) [reply Mac/BuildScript/resources/License.rtf:109]: Has this change been conformed with lawyers? This is a legal document and cannot be changed arbitrarily.

File: Misc/HISTORY [hunk 6]
```diff
@@ -31454,7 +31454,7 @@ aimed at a web server, checks that server for dead links.  Available
 are a command line utility as well as a Tkinter based GUI version.  In
 Tools/webchecker.  A simplified version of this program is dissected
 in my article in O'Reilly's WWW Journal, the issue on Scripting
-Languages (Vol 2, No 2); Scripting the Web with Python (pp 97-120).
+Languages (Vol 2, No 2); Scripting the web with Python (pp 97-120).
```
Comment (serhiy-storchaka) [inline Misc/HISTORY:31457]: It is the title of the book. It should not be changed until a new edition with changed title be published (and page numbers should be corrected in that case).

Comment (felixxm) [reply Misc/HISTORY:31457]: I didn't notice that it's a title :facepalm: , sorry.

**Comment by felixxm** [comment]
@terryjreedy @serhiy-storchaka Thanks for reviews :+1: I prepared PR with requested edits #27385.

**Comment by terryjreedy** [comment]
Some of us routinely try doc change backports.  If they do so cleanly, fine.  If not, optional -- if someone makes the new PR.

**Pull request merged**